make/dev.mk

```diff
@@ -68,4 +68,14 @@ add-member-to-host:
 .PHONY: add-host-to-member
 ## Run script add-cluster.sh host host-cluster
 add-host-to-member:
-	@${ADD_CLUSTER_SCRIPT_PATH} host host-cluster
\ No newline at end of file
+	@${ADD_CLUSTER_SCRIPT_PATH} host host-cluster
+
+.PHONY: deploy-csv
+## Create the test namespace
+deploy-csv: docker-push
+	sed -e 's|REPLACE_IMAGE|${IMAGE}|g' hack/deploy_csv.yaml | oc apply -f -
+
+.PHONY: install-operator
```

Comment:
It probably could depend on `deploy-csv`, right? So, `make install-operator` is enough to deploy the CSV and install it.

Reply:
done 0f85a44